Automattic/wp-calypso
# Bulk Plugin Management: Allow plugins menu to be contextual

Related to https://github.com/Automattic/dotcom-forge/issues/9700

## Proposed Changes

* Show certain menu items based on context eg. if plugin is active on all sites there is no need to show the `Activate` menu item.

## Why are these changes being made?


* Consistency and product quality

## Testing Instructions



* Ensure one of your plugin is enabled on all sites
* Go to `/plugins/manage`
* Click the action ellipses menu and verify the `Activate` menu item is not present, however you should see the `Deactivate` item.

![image](https://github.com/user-attachments/assets/41a3f154-ca66-49a7-8f38-5dab2ac0996f)


## Pre-merge Checklist
